dcat/index.html

```diff
@@ -4003,12 +4001,10 @@ <h2>Use of inverse properties</h2>
 <td id="inverse-of-catalog_catalog"><code>dcat:inCatalog</code></td>
 </tr>
 -->
-<!--
 <tr>
-<td><a href="#Property:resource_in_series"><code>dcat:inSeries</code></a></td>
+<td><a href="#Property:dataset_in_series"><code>dcat:inSeries</code></a></td>
 <td id="inverse-of-resource_in_series"><code>dcat:seriesMember</code></td>
```

Comment:
should this be ````inverse_of_dataset_in_series`````inverse-of-resource_in_series```

Reply:
[09d354f](https://github.com/w3c/dxwg/pull/1461/commits/09d354fe2cd2a98abf8c7a54476f202a0c62ad80) fixed this and  has made the rest of ids in the table consistent with  it replacing "inverse-of-" by "inverse_of_"

@agbeltran  can you give your ok to this PR?